pkg/csource/csource_test.go

```diff
@@ -330,3 +330,51 @@ func TestGenerateSandboxFunctionSignature(t *testing.T) {
 		"do_sandbox_android(-1234);", // expected
 		"Android sandbox function requires an argument")
 }
+
+func TestCommentGeneration(t *testing.T) {
+	progString := `r0 = socket$netlink(0x10, 0x3, 0x2)
+bind$netlink(r0, &(0x7f0000514ff4)={0x10, 0x0, 0x0, 0x2ffffffff}, 0xc)`
+
+	expectedComments := []string{
+		`//  socket$netlink arguments: [
+//    const: 0x10
+//    const: 0x3
+//    netlink_proto: 0x2
+//  ]
+`,
+		`//  bind$netlink arguments: [
+//    *ptr[in, sockaddr_nl_t[AF_NETLINK, netlink_port_id,
+//    flags[netlink_group_bitmap, int32]]]: {
+//      sockaddr_nl_t[AF_NETLINK, netlink_port_id, flags[netlink_group_bitmap,
+//      int32]] {
+//        const: 0x10
+//        const: 0x0
+//        int32: 0x0
+//        netlink_group_bitmap: 0x2ffffffff
+//      }
+//    }
+//    len: 0xc
+//  ]
+`}
+	target, err := prog.GetTarget(targets.Linux, targets.AMD64)
+	if err != nil {
+		t.Fatalf("%v", err)
+	}
+
+	p, err := target.Deserialize([]byte(progString), prog.NonStrict)
```

Comment:
It's better to use Strict mode, otherwise these tests will get out of sync with descriptions (which will likely to fail the test anyway, but the failure mode will be obscure).